app/app.js

```diff
@@ -139,3 +153,39 @@ function mergeObjects(og, so) {
     }
     return og;
 }
+
+function areDiffBlocksCollapsed()
```

Comment:
Could add the `{` at the same line (`function areDiffBlocksCollapsed() {`), just to keep the code consistency :)

Reply:
Totally.